app.js

```diff
@@ -64,11 +64,12 @@ for(let i = 0; i < showSamples.length; i++) {
 main.insertAdjacentHTML('beforeend', html1);
 
 //Fetch Functions ---------------------------------------------------------------------------------------------------------------------
-function fetchInfo(url) {
+async function fetchInfo(url) { // Comment on async
     return fetch(url)
         .then(response => response.json())
 }
 
+// Comment on dead code
```

Comment:
Leaving a large block of commented-out code is a [code smell](https://en.wikipedia.org/wiki/Code_smell).  You should remove it.  Git will remember it was here if you ever need to find it again. 

To find a random string you can either use the Github search bar or you can use the git command like `git log -S fetchJSON`.  That command will show you the commit where that string is seen.  You can then copy the commit ID `ceb9c5fdeb239c027481b9563f24e173fbafa965` and use `git show ceb9c5fdeb239c027481b9563f24e173fbafa965` to see what all changed in that commit.